### ISSUE TITLE
Add option to replace with rem instead of em

### ISSUE THREAD

**Issue by rossjrw** [chiedojohn/vim-px-to-em#1]
`rem` is the root font size and is always 16px, unless the user has changed their browser's font size setting.

`em` is the scoped font size, and is equal to the root font size unless it's been changed somewhere in the tree.

If I did the following:

```html
<div>
  <p class="text1">One</p>
  <div class="smaller">
    <p class="text2">Two</p>
  </div>
</div>
```

```css
.text1, .smaller, .text2 {
  font-size: 0.5em;
}
```

Then the font size of `.text1` would be 8px, and the font size of `.text2` would be 4px - because the 0.5em is a relative unit, and has been applied twice.

Whereas if I instead did `font-size: 0.5rem`, both `p`s would be 8px, because rem is an absolute unit.

In summary: replacement of px to em must be context-aware. rem is absolute and universal, so this plugin should be using rem instead of em.
